src/utils/ui-utils.ts

```diff
@@ -316,4 +316,46 @@ export function convertVWToPixel(value: number) {
 
   const result = (x * value) / 100;
   return result;
+}
```

Comment:
Let's move all these cookie related functions to a separate file and call it `cookie.ts` and move it to `services` folder.

And the file should define a class and these functions that you defined here should be static properties of it. Something like

```
export default class CookieService {

  static deleteCookie(name:string, path = '/', domain?:string) {
   ...
  }
  
 ...
}
```